runtime/parachains/src/paras_inherent.rs

```diff
@@ -563,9 +576,6 @@ impl<T: Config> Pallet<T> {
 						e
 					});
 
-				// current concluded invalid disputes, only including the current block's votes
```

Comment:
This comment seems to have disappeared. The question below about phrasing is relevant, and the best answer I can come up with is that it should be rephrased to "current concluded invalid disputes, only including the current _session's_ disputes"

Reply:
The reason for that, by the way, is because it lightens the load on `collect_disputed` - cores can't be occupied with things from previous sessions, and only things with new votes can have just concluded. We only need to collect cores with 'just concluded' disputes, because disputes that concluded longer ago have already had any corresponding cores cleaned up.